Document/0x06d-Testing-Data-Storage.md

```diff
@@ -315,55 +299,124 @@ $ grep -iRn keyword .
+Save the output by adding `--json keychain.json` to this command
+Dumping the iOS keychain...
+Created                    Accessible                      ACL    Type      Account                    Service                                                        Data
+-------------------------  ------------------------------  -----  --------  -------------------------  -------------------------------------------------------------  ------------------------------------
+2020-02-11 13:26:52 +0000  WhenUnlocked                    None   Password  keychainValue              com.highaltitudehacks.DVIAswiftv2.develop                      mysecretpass123
 ```  
 
 ###### Searching for Binary Cookies
 
-iOS applications often store binary cookie files in the application sandbox. Cookies are binary files containing cookie data for application WebViews. You can use Needle to convert these files to a readable format and inspect the data. Use the following Needle module, which searches for binary cookie files stored in the application container, lists their data protection values, and gives the user the options to inspect or download the file:
+iOS applications often store binary cookie files in the application sandbox. Cookies are binary files containing cookie data for application WebViews. You can use Objection to convert these files to a json format and inspect the data.
 
 ```bash
-[needle] > use storage/data/files_binarycookies
-[needle][files_binarycookies] > run
+...itudehacks.DVIAswiftv2.develop on (iPhone: 13.2.3) [usb] # ios cookies get --json
+[
+    {
+        "domain": "highaltitudehacks.com",
+        "expiresDate": "2051-09-15 07:46:43 +0000",
+        "isHTTPOnly": "false",
+        "isSecure": "false",
+        "name": "username",
+        "path": "/",
+        "value": "admin123",
+        "version": "0"
+    }
+]
 ```
 
 ###### Searching for Property List Files
 
-iOS applications often store data in property list (plist) files that are stored in both the application sandbox and the IPA package. Sometimes these files contain sensitive information, such as usernames and passwords; therefore, the contents of these files should be inspected during iOS assessments. Use the following Needle module, which searches for plist files stored in the application container, lists their data protection values, and gives the user the options to inspect or download the file:
+iOS applications often store data in property list (plist) files that are stored in both the application sandbox and the IPA package. Sometimes these files contain sensitive information, such as usernames and passwords; therefore, the contents of these files should be inspected during iOS assessments. Use the *ios plist cat plistFileName.plist* command to inspect the plist file.
+
+To find userInfo.plist find use the *env* command. It will print out the locations of the applications Library, Caches and Documents directories:
```

Comment:
```suggestion
To find the file userInfo.plist, use the `env` command. It will print out the locations of the applications Library, Caches and Documents directories:
```